### ISSUE TITLE
`canStale` should return false if `Cache-Control: must-revalidate` is present

### ISSUE THREAD

**Issue by arthurfiorette** [arthurfiorette/axios-cache-interceptor#507]
### Describe the bug

The `canStale` function is used by the buildStorage function to determine when the cache can become `state: 'stale'` or not, but `must-revalidate` is not being respected.

**Currently, mustRevalidate disables the cache, which is a bug**

### To Reproduce

```ts
import { canStale } from 'axios-cache-interceptor';

const { id } = await axios.get('url to a server that returns must-revalidate and etag');
const storage = await axios.storage.get(id);

canStale(storage) === true;
```

### Expected behavior

`canStale()` should return false when `must-revalidate` is present in `Cache-Control`

### Additional context

- Axios: `v1.3.1`
- Axios Cache Interceptor: `v1.0.1`

**Comment by arthurfiorette** [comment]
Anyone up to working on this?

**Comment by rishisundar** [comment]
Hey! @arthurfiorette, can I work on this?

**Comment by arthurfiorette** [comment]
Of couse! I'd start removing the `mustRevalidate` check on the default header interpreter and updating the `canStale` function.

**Comment by arthurfiorette** [comment]
Implemented by 22870d7.
